lib/compat/wordpress-6.9/site-editor-redirect.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
I took this pattern from 

https://github.com/WordPress/gutenberg/blob/8a4cb319deb51bf2feb7ac632b6b24ae65261b2f/lib/compat/wordpress-6.8/site-editor.php#L28